.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,58 @@
+name: 🔓 Feature Request
+description: File a feature request
+title: '[Feature]: '
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for wanting to contribute to this repository. 
+
+        Before you do, please ensure you are filing the ticket in the right
+        place. 
+
+        If you have a question use
+        [Discussions](https://github.com/fabricjs/fabric.js/discussions).
+
+        If you are up for PRing see [Pull Requests](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#-pull-requests). 
```

```suggestion
        If you are interested in submitted a PR see [Pull Requests](https://github.com/fabricjs/fabric.js/blob/master/CONTRIBUTING.md#-pull-requests).         
```

We should probably stick to clear English for the benefit of google translate users